### ISSUE TITLE
package.json requires a nodejs not supported by current nodejs buildpack

### ISSUE THREAD

**Issue by gberche-orange** [cloudfoundry-community/etherpad-lite-cf#7]
```
017-08-07T15:07:48.70+0200 [STG/0] OUT Destroying container
2017-08-07T15:07:49.61+0200 [STG/0] OUT Successfully destroyed container
2017-08-07T15:16:21.58+0200 [API/0] OUT Creating build for app with guid ca5962f5-9305-4630-b44d-a0078d0e89a2
2017-08-07T15:16:21.67+0200 [API/0] OUT Updated app with guid ca5962f5-9305-4630-b44d-a0078d0e89a2 ({"state"=>"STARTED"})
2017-08-07T15:16:21.69+0200 [STG/0] OUT Downloading nodejs_buildpack...
2017-08-07T15:16:21.71+0200 [STG/0] OUT Downloaded nodejs_buildpack
2017-08-07T15:16:21.71+0200 [STG/0] OUT Creating container
2017-08-07T15:16:22.34+0200 [STG/0] OUT Successfully created container
2017-08-07T15:16:22.35+0200 [STG/0] OUT Downloading app package...
2017-08-07T15:16:26.09+0200 [STG/0] OUT Downloaded app package (15.2M)
2017-08-07T15:16:26.09+0200 [STG/0] OUT Downloading build artifacts cache...
2017-08-07T15:16:26.28+0200 [STG/0] OUT Downloaded build artifacts cache (398.9K)
2017-08-07T15:16:26.28+0200 [STG/0] OUT Staging...
2017-08-07T15:16:27.00+0200 [STG/0] OUT -------> Buildpack version 1.5.34
2017-08-07T15:16:27.01+0200 [STG/0] OUT -----> Installing binaries
2017-08-07T15:16:27.03+0200 [STG/0] OUT        engines.node (package.json):  ~5.11.0
2017-08-07T15:16:27.03+0200 [STG/0] OUT        engines.npm (package.json):   unspecified (use default)
2017-08-07T15:16:27.03+0200 [STG/0] OUT        
2017-08-07T15:16:27.53+0200 [STG/0] OUT        Downloading and installing node ...
2017-08-07T15:16:28.01+0200 [STG/0] ERR DEPENDENCY MISSING IN MANIFEST: node 
2017-08-07T15:16:28.01+0200 [STG/0] ERR It looks like you're trying to use node .
2017-08-07T15:16:28.01+0200 [STG/0] ERR Unfortunately, that version of node is not supported by this buildpack.
2017-08-07T15:16:28.01+0200 [STG/0] ERR The versions of node supported in this buildpack are:
2017-08-07T15:16:28.01+0200 [STG/0] ERR - 7.9.0
2017-08-07T15:16:28.01+0200 [STG/0] ERR - 7.10.0
2017-08-07T15:16:28.01+0200 [STG/0] ERR - 6.10.3
2017-08-07T15:16:28.01+0200 [STG/0] ERR - 6.10.2
2017-08-07T15:16:28.01+0200 [STG/0] ERR - 4.8.3
2017-08-07T15:16:28.01+0200 [STG/0] ERR - 4.8.2
2017-08-07T15:16:28.01+0200 [STG/0] ERR If you need further help, start by reading: https://github.com/cloudfoundry/nodejs-buildpack/releases.
2017-08-07T15:16:28.01+0200 [STG/0] OUT -----> Build failed
2017-08-07T15:16:28.01+0200 [STG/0] OUT        
2017-08-07T15:16:28.01+0200 [STG/0] OUT        We're sorry this build is failing! You find more info about the nodejs buildpack here:
2017-08-07T15:16:28.01+0200 [STG/0] OUT        https://docs.cloudfoundry.org/buildpacks/node/index.html
2017-08-07T15:16:28.01+0200 [STG/0] OUT        

```

**Comment by o-orand** [comment]
Etherpad lite version 1.6.1 (see [release note](https://github.com/ether/etherpad-lite/releases)) is required to fix node 7 compatibility issue

**Comment by gberche-orange** [comment]
Upstream upgrade procedure is documented at https://github.com/cloudfoundry-community/etherpad-lite-cf/blob/cloudfoundry/UPDATE.md

**Comment by gberche-orange** [comment]
Unsuccessfully tried merging upstream 1.6.1 with previous node7 upgrade from @poblin-orange in [update_1_6_1](https://github.com/cloudfoundry-community/etherpad-lite-cf/compare/cloudfoundry...cloudfoundry-community:update_1_6_1?expand=1) branch.

Failed in https://travis-ci.org/cloudfoundry-community/etherpad-lite-cf/builds/263540878 with

> $ mv node_modules/ep_etherpad-lite/node_modules/ep_* node_modules
> mv: cannot stat `node_modules/ep_etherpad-lite/node_modules/ep_*': No such file or directory

Would need local building and checking upstream changes. 
Moving this into a distinct issue: #9

**Comment by gberche-orange** [comment]
The prerelease https://github.com/cloudfoundry-community/etherpad-lite-cf/releases/tag/1.6.0.2-cf failed at start up with 

```
2017-08-18T17:05:36.37+0200 [CELL/0] OUT Starting health monitoring of container
2017-08-18T17:05:36.69+0200 [APP/PROC/WEB/0] OUT Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.
2017-08-18T17:05:36.70+0200 [APP/PROC/WEB/0] OUT Ensure jQuery is downloaded and up to date...
2017-08-18T17:05:36.70+0200 [APP/PROC/WEB/0] OUT Clearing minified cache...
2017-08-18T17:05:36.70+0200 [APP/PROC/WEB/0] OUT Ensure custom css/js files are created...
2017-08-18T17:05:36.70+0200 [APP/PROC/WEB/0] OUT Started Etherpad...
2017-08-18T17:05:36.70+0200 [APP/PROC/WEB/0] OUT Started Etherpad...
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR module.js:472
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     throw err;
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     ^
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR Error: Cannot find module 'internal/fs'
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Function.Module._resolveFilename (module.js:470:15)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Function.Module._load (module.js:418:25)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Module.require (module.js:498:17)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at require (internal/module.js:20:19)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at evalmachine.<anonymous>:18:20
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Object.<anonymous> (/home/vcap/app/src/node_modules/npm/node_modules/graceful-fs/fs.js:11:1)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Module._compile (module.js:571:32)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Object.Module._extensions..js (module.js:580:10)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at Module.load (module.js:488:32)
2017-08-18T17:05:36.85+0200 [APP/PROC/WEB/0] ERR     at tryModuleLoad (module.js:447:12)
```

which corresponds to https://github.com/ether/etherpad-lite/issues/3074 fixed in 1.6.1

**Comment by gberche-orange** [comment]
Failed to upgrade to nodejs 7 and etherpad-lite 1.6.1 in https://github.com/cloudfoundry-community/etherpad-lite-cf/issues/9#issuecomment-326335341

Temporary fallback is to stick to 1.6.0 and upgrade to nodejs 6.9 whose end of life is april 2019 according to https://github.com/nodejs/LTS. This is packaged into release 1.6.0-cf.2 from https://github.com/cloudfoundry-community/etherpad-lite-cf/commit/7e240b607b499e06c9dc46002bc8de9264cf3201
